app/services/edit_draft_edition_service.rb

```diff
@@ -2,9 +2,9 @@
 
 class EditDraftEditionService < ApplicationService
   def initialize(edition:, user:, **attributes)
-    @edition = edition
-    @user = user
-    @attributes = attributes
+    super(edition: edition,
+          user: user,
+          attributes: attributes)
```

Comment:
You hint in the commit message that this was necessary, but could you explain a bit more in the commit about why it's necessary?

I guess it's because we want any additional arbitrary values to come under an `attributes` key in this one case. I wonder if it's worth changing the `EditDraftEditionService` to look for attributes under the top level rather than under `attributes`, in which case this special behaviour wouldn't be necessary?